filebeat/scripts/docs_collector.py

```diff
@@ -1,6 +1,7 @@
 import os
 import argparse
 import yaml
+import six
```

Comment:
There are two instances of `six.iteritems` in that file ([here](https://github.com/lukebakken/beats/blob/fixes/lrb/gh-3166-python-3-dict/filebeat/scripts/docs_collector.py#L63) and [here](https://github.com/lukebakken/beats/blob/fixes/lrb/gh-3166-python-3-dict/filebeat/scripts/docs_collector.py#L68))